src/Snap/Snaplet/#SqliteSimple.hs#

```diff
@@ -0,0 +1,227 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
No need for a separate PR.  You should be able to use "git rebase -i", clean things up as you desire, and then "git push -f" the same topic into this PR.  

If you've never done that, might want to be careful with doing that.